### PR TITLE
Create List Screen

### PR DESCRIPTION
**Pull request by VladimirGnatovsky**
ListScreen

### CODE REVIEW

File: src/modules/types.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name: string;
+  description: string;
+  image: any;
+};
+
+export type FilmsT = {
+  image: any;
+  lastDescription: string;
+  name: string;
+};
```
Comment (Ragnarreck) [inline src/modules/types.tsx:16]: Not the best way to add `T` like a type to type name. Also, you describe only one `Film` here, not the array.

File: src/components/ListFilms.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const ListFilms = ({ image, lastDescription, name }: FilmsT) => (
+  <ScrollView>
+    <View style={styles.containerMessage}>
+      <Image source={image} style={styles.avatar} />
+      <View>
+        <Text style={styles.text}>{name}</Text>
+        <Text style={styles.message}>{lastDescription}</Text>
+      </View>
+    </View>
+  </ScrollView>
```
Comment (Ragnarreck) [inline src/components/ListFilms.tsx:15]: Doesn't seem like it's a list, more like a single item?

File: src/modules/list/ListScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+import DEMO from '../../../assets/data/demo';
+import theme from '../../constants/Color';
+
+
+const ListScreen = () => {
+  return (
+    <Background>
+      <Logo />
+      <View style={styles.boxShadow}>
+        <Header>Movies</Header>
```
Comment (Ragnarreck) [inline src/modules/list/ListScreen.tsx:21]: `FlatList` has a property called `ListHeaderComponent` which you can use for your purpose

File: src/modules/types.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export type Navigation = {
+  navigate: (scene: string) => void;
+};
+
+export type DataT = {
```
Comment (Ragnarreck) [inline src/modules/types.tsx:5]: Same here

**Review by danilaka25 — APPROVED** [review]

**Pull request closed without merging**